.vscode/launch.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
Looks like these files in `.vscode/` slipped through the cracks, use `git rm -r --cached <folder>` to remove them and commit again. I've had this issue before, I think it has to do with git only ignoring adding new files, not ones that have already been added to a commit.